README.md

```diff
@@ -1,29 +1,106 @@
 
-## Getting started
+## Requirements
 
-- Clone the project locally
-- Optionally checkout a specific version, for example: `git checkout tags/v0.1`
-- Start your environment and build Drupal
-- Login to Drupal in your browser and copy the session cookie from the browser
-- Run a benchmark with ab (in Ubuntu install with `sudo apt-get install apache2-utils`)
+- Apache Benchmark (ab) tool
+  - Ubuntu/Debian: `sudo apt-get install apache2-utils`
+  - macOS: Included with macOS
+- Either [DDEV](https://ddev.com/) or [Lando](https://lando.dev/)
+- curl
+- bc
 
-This project comes with [Lando](https://lando.dev/) and [DDEV](https://ddev.com/) environments preconfigured. Please find more detailed examples for running the benchmark in these environments below.
+## Getting Started
 
-### Lando
+1. Clone the project locally
+2. Optionally checkout a specific version: `git checkout tags/v0.1`
+3. Start your environment and install Drupal using one of the methods below
 
+### DDEV Setup
 ```sh
-lando start && lando drush si --yes --existing-config
+ddev start
+ddev composer install
+ddev drush si --yes --existing-config
+./benchmark.sh ddev
+```
+
+### Lando Setup
+```sh
+lando start
+lando drush si --yes --existing-config
 ./benchmark.sh lando
 ```
 
-### DDEV
+## Running Benchmarks
+
+The benchmark script supports several parameters:
+
+```sh
+./benchmark.sh <ddev|lando> [concurrent] [requests] [urls_file]
+```
+
+Parameters:
+- `ddev|lando`: Required. Specifies which local development environment to use
+- `concurrent`: Optional. Number of concurrent users (default: 10)
+- `requests`: Optional. Total number of requests per URL (default: 2000)
+- `urls_file`: Optional. Path to file containing URLs to test (default: urls.txt)
 
+Examples:
 ```sh
-ddev start && ddev composer install && ddev drush si --yes --existing-config
+# Basic usage
 ./benchmark.sh ddev
+
+# Custom concurrent users (20) and requests (1000)
+./benchmark.sh lando 20 1000
+
+# Using a custom URLs file
+./benchmark.sh ddev 10 500 my-custom-urls.txt
+```
+
+## URLs Configuration
+
+By default, the script tests these URLs:
+- /admin/modules
+- /admin/config
+- /admin/content
+- /node/add
+- /admin/people
+
+You can create a custom URLs file to test different paths:
+```txt
+# my-custom-urls.txt
+/admin/modules
+/node/1
+/user
+# Lines starting with # are ignored
+```
+
+## Results
+
+Benchmark results are saved in a timestamped directory `benchmark_results_YYYYMMDD_HHMMSS` containing:
+- Individual test results for each URL
+- A summary file with key metrics including:
+  - Requests per second
+  - Mean time per request
+  - Failed requests count
+  - Total execution time
+
+## Performance Tips
+
+### DDEV Performance
+[DDEV comes with Mutagen for file syncing built in](https://ddev.readthedocs.io/en/latest/users/performance/), which can make an environment on macOS multiple times faster. You can enable Mutagen for your local DDEV globally with:
+```sh
+ddev config global --mutagen-enabled
```

Comment:
I think this info is outdated. Nowadays you would need to set the [performance_mode](https://ddev.readthedocs.io/en/stable/users/configuration/config/#performance_mode) setting. This has been the case since [ddev v1.22.0](https://github.com/ddev/ddev/releases/tag/v1.22.0).